library.properties

```diff
@@ -2,7 +2,7 @@ name=tiny-AES-c
 version=1.0.0
 author=kokke
 maintainer=kokke
-sentence=A small and portable implementation of the AES ECB, CTR and CBC encryption algorithms written in C.
+sentence=A small and portable implementation of the AES ECB, CTR, CMAC and CBC encryption algorithms written in C.
```

Comment:
Small nitpick:

`implementation of the AES ECB, CTR, **CMAC** and CBC **encryption algorithms** written in C.` - emphasis mine.

CMAC is not an encryption algorithm but a MAC - the same applies to the change you've made in README.